dir.props

```diff
@@ -1,11 +1,13 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
   <Import Project="src\BuildValues.props" />
-  
+
   <!--
     $(OS) is set to Unix/Windows_NT. This comes from an environment variable on Windows and MSBuild on Unix.
   -->
   <PropertyGroup>
+    <!-- Temp change to make OS X build behave as a Linux build -->
+    <OsEnvironment Condition="'$(OsEnvironment)'=='' AND '$(OS)'=='OSX'">Unix</OsEnvironment>
```

Comment:
Wow -- I don't know the reason for this temp change in CoreFx, but it looks like it will block attempts to have OSX-specific code, as I believe is needed for crypto.  Don't need to change this PR.  The CoreFx repo needed an issue number here.